uat/testing-features/src/main/resources/greengrass/features/ggad-1.feature

```diff
@@ -0,0 +1,96 @@
+@GGAD
+Feature: GGAD-1
```

Comment:
Could you rename GGAD abbreviations, please to GGMQ and merge test scenarios in a single file? ggmq-1.feature

Reply:
fixed